tests/global/tglobal.nim

```diff
@@ -1,6 +1,6 @@
 discard """
   output: "in globalaux2: 10\ntotal globals: 2\nint value: 100\nstring value: second"
-  disabled: "true"
+    disabled: "true"
```

Comment:
Disabled around 2015-03, for CI sake?
Working on my pc.

Reply:
Enable